src/ducks/posts.ts

```diff
@@ -230,6 +225,22 @@ export const fetchLikedBy =
     return json.payload.map((post: any) => post.messageId);
   };
 
+export const fetchLikersByPost = async (
```

Comment:
This bypass/doesn't use `redux` (no use of dispatch). Is that ok?

Reply:
i would make it a redux action just so that it's consistent with other calls.

alternatively, you can also just put this call inside the PostLikes modal, which is ideally where we want to make the request